### PR TITLE
LRQA-72732 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-72732

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
Hi @melodywu, this is not very different from other tests that filters by Override. Can you expand this test case and make it more unique and be all about filtering Overrides? What do you see when you filter Overrides that you cannot see in other pages? You would want to assert those elements. Maybe you would also want to assert that no existing keys display when you filter Overridden keys.

In addition, to save time, if we are just filtering by overrides, we don't need to add a language key with 4 translation overrides since that is already covered in other tests.

**Comment by melodywu** [comment]
> Hi @melodywu, this is not very different from other tests that filters by Override. Can you expand this test case and make it more unique and be all about filtering Overrides? What do you see when you filter Overrides that you cannot see in other pages? You would want to assert those elements. Maybe you would also want to assert that no existing keys display when you filter Overridden keys.
> 
> In addition, to save time, if we are just filtering by overrides, we don't need to add a language key with 4 translation overrides since that is already covered in other tests.

Done

**Comment by melodywu** [comment]
ci:test:sf

File: modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverride.testcase [hunk 1]
```diff
@@ -53,6 +53,60 @@ definition {
...
+
+			AssertTextEquals(
+				key_languageKey = "0-analytics-cloud-connection",
+				locator1 = "LanguageOverride#VIEW_PAGE_LANGUAGE_KEY",
+				value1 = "0-analytics-cloud-connection");
+
+			LanguageOverride.addLanguageKey(
+				languageKey = "text-contains-error",
+				translationOverrideList = "en-US",
+				translationOverrideValueList = "Text contains error");
```
Comment (patriciajeanperez) [inline modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverride.testcase:80]: @melodywu I think this should have been under the next task?

File: modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverride.testcase [hunk 2]
```diff
@@ -53,6 +53,60 @@ definition {
...
+				languageKey = "text-contains-error",
+				overridded = "true",
+				translationOverrideList = "en-US",
+				translationOverrideValueList = "Text contains error update");
+
+			AssertTextEquals(
+				key_translation = "Text contains error update",
+				locator1 = "LanguageOverride#VIEW_PAGE_LANGUAGE_KEY_TRANSLATION",
+				value1 = "Text contains error update");
+		}
```
Comment (patriciajeanperez) [inline modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverride.testcase:107]: @melodywu why did we have to update the language key for this test?

Comment (melodywu) [reply modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverride.testcase:107]: I want to check the override button can be click as edit button of key

File: modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverride.testcase (hunk 1 above)
Comment (melodywu) [reply modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverride.testcase:80]: OK I will remove it

**Comment by melodywu** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
Hi @melodywu, I noticed that tasks aren't updated when you make changes to the macros inside it. Please make sure to update those too so they are accurate with what the task does. Please see my additional changes in https://github.com/patriciajeanperez/liferay-portal/pull/564. Thanks!

**Comment by melodywu** [comment]
> Hi @melodywu, I noticed that tasks aren't updated when you make changes to the macros inside it. Please make sure to update those too so they are accurate with what the task does. Please see my additional changes in #564. Thanks!

Thanks

**Pull request closed without merging**